charts/islandis/values.staging.yaml

```diff
@@ -223,16 +233,10 @@ application-system-api:
     DOCUMENT_PROVIDER_ONBOARDING_REVIEWER: '/k8s/application-system/api/DOCUMENT_PROVIDER_ONBOARDING_REVIEWER'
     DOKOBIT_ACCESS_TOKEN: '/k8s/application-system/api/DOKOBIT_ACCESS_TOKEN'
     DOKOBIT_URL: '/k8s/application-system-api/DOKOBIT_URL'
-    DRIVING_LICENSE_SECRET: '/k8s/application-system/api/DRIVING_LICENSE_SECRET'
```

Comment:
Isn't `XROAD_DRIVING_LICENSE` missing? Same for prod and dev files. Added another comment on the corresponding infra file for `application-system-api`

Reply:
whew nice catch. added. thanks!